ethereum-consensus/src/capella/block_processing.rs

```diff
@@ -1,12 +1,15 @@
 use crate::{
     capella::{
-        compute_timestamp_at_slot, get_current_epoch, get_randao_mix, process_attestation,
-        process_attester_slashing, process_block_header, process_deposit, process_eth1_data,
-        process_proposer_slashing, process_randao, process_sync_aggregate, process_voluntary_exit,
-        BeaconBlock, BeaconBlockBody, BeaconState, ExecutionEngine, ExecutionPayload,
-        ExecutionPayloadHeader, NewPayloadRequest, SignedBlsToExecutionChange,
+        compute_domain, compute_timestamp_at_slot, get_current_epoch, get_randao_mix,
+        process_attestation, process_attester_slashing, process_block_header, process_deposit,
+        process_eth1_data, process_proposer_slashing, process_randao, process_sync_aggregate,
+        process_voluntary_exit, BeaconBlock, BeaconBlockBody, BeaconState, BlsPublicKey,
+        DomainType, ExecutionEngine, ExecutionPayload, ExecutionPayloadHeader, NewPayloadRequest,
+        SignedBlsToExecutionChange,
     },
-    ssz::prelude::*,
+    primitives::{BLS_WITHDRAWAL_PREFIX, ETH1_ADDRESS_WITHDRAWAL_PREFIX},
+    signing::verify_signed_data,
+    ssz::{ByteVector,prelude::*},
```

Comment:
went ahead and changed this so we can just do this
```suggestion
    ssz::prelude::*,
```

Reply:
should have checked before committing it , but ByteVector isnt inn the prelude, thats why CI is failing